storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/SnowflakeQueryTemplater.java

```diff
@@ -146,22 +141,16 @@ protected String createJoinQuery(
     return writer.toString();
   }
 
-  // TODO: export as decoded csv file
   @Override
-  protected List<String> generateExportTableSqlQuery(String resultTable, String stagingPath) {
-    String exportPath = String.format("%s/", stagingPath.replaceAll("/$", ""));
+  protected List<String> generateExportTableSqlQuery(String resultTable, String stagingUri) {
 
     List<String> exportTableSqlQueries = new ArrayList<>();
-    String createStageQuery = String.format("create or replace stage my_stage;");
     String copyIntoStageQuery =
         String.format(
-            "COPY INTO '@my_stage/%s.%s' FROM %s file_format = (type=csv compression='gzip')\n"
-                + "single=true header = true;",
-            resultTable, EXPORT_FILE_FORMAT, resultTable);
-    String downloadTableQuery =
-        String.format(
-            "get @my_stage/%s.%s file://%s;", resultTable, EXPORT_FILE_FORMAT, exportPath);
-    String[] queryArray = new String[] {createStageQuery, copyIntoStageQuery, downloadTableQuery};
+            "COPY INTO '%s%s.%s' FROM %s file_format = (type=csv compression='gzip')\n"
```

Comment:
staging_location should support "s3://feast-snowflake-staging/test/" and "s3://feast-snowflake-staging/test"